x-pack/plugins/security_solution/public/assistant/helpers.tsx

```diff
@@ -61,7 +81,14 @@ const sendToTimelineEligibleQueryTypes: Array<CodeBlockDetails['type']> = ['kql'
 export const augmentMessageCodeBlocks = (
   currentConversation: Conversation
 ): CodeBlockDetails[][] => {
-  const cbd = currentConversation.messages.map(({ content }) => analyzeMarkdown(content));
+  const cbd = currentConversation.messages.map(({ content }) =>
```

Comment:
fixes an issue where the UUIDs instead of the original values were added to the filter / kql / eql when adding query output to a timeline